src/index.ts

```diff
@@ -4,47 +4,44 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import {NativeModules} from 'react-native';
+import { NativeModules } from 'react-native';
 
-const {RNCImageEditor} = NativeModules;
+const { RNCImageEditor } = NativeModules;
 
-type ImageCropData = {
+type $Maybe<T> = T | null | undefined;
+
+export interface ImageCropData {
   /**
    * The top-left corner of the cropped image, specified in the original
    * image's coordinate space.
    */
-  offset: {|
-    x: number,
-    y: number,
-  |},
+  offset: {
+    x: number;
+    y: number;
+  };
   /**
    * The size (dimensions) of the cropped image, specified in the original
    * image's coordinate space.
    */
-  size: {|
-    width: number,
-    height: number,
-  |},
+  size: {
+    width: number;
+    height: number;
+  };
   /**
    * (Optional) size to scale the cropped image to.
    */
-  displaySize?: ?{|
-    width: number,
-    height: number,
-  |},
+  displaySize?: $Maybe<{
+    width: number;
+    height: number;
+  }>;
   /**
    * (Optional) the resizing mode to use when scaling the image. If the
    * `displaySize` param is not specified, this has no effect.
    */
-  resizeMode?: ?$Enum<{
-    contain: string,
-    cover: string,
-    stretch: string,
-  }>,
-};
+  resizeMode?: $Maybe<'contain' | 'cover' | 'stretch'>;
```

Comment:
`$Maybe` seems unnecessary. `?` should already allow passing `undefined`.

Reply:
right, will be fixed in flowing PR (new arch)